src/table/create.rs

```diff
@@ -266,6 +267,49 @@ impl TableCreateStatement {
         self.indexes.as_ref()
     }
 
+    /// Rewriting extra param. You should take care self about concat extra params. Add extra after options.
+    /// Example for PostgresSQL [Citus](https://github.com/citusdata/citus) extension:
+    /// ```
+    /// use sea_query::{*, tests_cfg::*};
+    /// let table = Table::create()
+    ///     .table(Char::Table)
+    ///     .col(
+    ///         ColumnDef::new(Char::Id)
+    ///             .uuid()
+    ///             .extra("DEFAULT uuid_generate_v4()".into())
+    ///             .primary_key()
+    ///             .not_null(),
+    ///     )
+    ///     .col(
+    ///         ColumnDef::new(Char::CreatedAt)
+    ///             .timestamp_with_time_zone()
+    ///             .extra("DEFAULT NOW()".into())
+    ///             .not_null(),
+    ///     )
+    ///     .col(ColumnDef::new(Char::UserData).json_binary().not_null())
+    ///     .set_extra(&"USING columnar")
+    ///     .to_owned();
+    /// assert_eq!(
+    ///     table.to_string(PostgresQueryBuilder),
+    ///     [
+    ///         r#"CREATE TABLE "character" ("#,
+    ///         r#""id" uuid DEFAULT uuid_generate_v4() PRIMARY KEY NOT NULL,"#,
+    ///         r#""created_at" timestamp with time zone DEFAULT NOW() NOT NULL,"#,
+    ///         r#""user_data" jsonb NOT NULL"#,
+    ///         r#") USING columnar"#,
+    ///     ]
+    ///     .join(" ")
+    /// );
+    /// ```
+    pub fn set_extra(&mut self, extra: &str) -> &mut Self {
```

Comment:
Do you think naming it `extra` sounds better? And we can take an `Into<String>` here.

```suggestion
    pub fn extra<T>(&mut self, extra: T) -> &mut Self
    where
        T: Into<String>,
    {
```

Reply:
Generic type is great, i just forgot about this.
Naming _set_ chosen for indicating rewriting extra param. If you want - ok, just name this _extra_.
Say me you decsion and i commit changes. 
Thank you for review